page_analyzer/utils/db.py

```diff
@@ -86,24 +86,17 @@ def get_urls_data() -> List[Dict[str, Any]]:
     return actual_urls_data
 
 
-def generate_check_context(url_id: int) -> Dict[str, int | str] | None:
+def generate_check_context(url_id: int) -> Dict[str, str] | None:
     url_context = get_url_context(url_id)
     if not url_context:
         return None
     url_name = url_context["name"]
     status_code = get_url_status_code(str(url_name))
     if status_code is None:
         return None
-    h1 = ""
-    title = ""
-    description = ""
-    check_context: dict[str, int | str] = {
-        "response_code": status_code,
-        "h1": h1,
-        "title": title,
-        "description": description,
-    }
-    return check_context
+    page_parsing_result: Dict[str, str] = parse_page_data(str(url_name))
```

Comment:
it's not necessary to declare types for something that gets its value from typed function

```suggestion
    page_parsing_result = parse_page_data(str(url_name))
```